CONTRIBUTING.md

```diff
@@ -24,7 +24,10 @@ For more information, please see STYLEGUIDE.md
 
 # <a name="Attachments"></a>Attachments
 
-GitHub only allows image attachments. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
-Remember to include a link to the attachment in your post here.
+GitHub only allows a few attachment extensions: .png, .gif, .jpg, .docx, .pptx, .xlsx, .txt, .pdf, .zip and .gz.  
```

Comment:
@SubJunk I don't find easily the information concerning the supported files formats, do you ?
I do not expect that people want search that much for such link:
https://github.com/blog/2061-attach-files-to-comments

So i don't think delete this comment should be the thing to do, and how many times did they changed they support ? the last time they added was in october 2015, and as they just add and not withdraw formats support, AFAIK, i think that should be kept as it is, IMO.